packages/main/cypress/specs/Panel.cy.tsx

```diff
@@ -0,0 +1,474 @@
+import Label from "../../src/Label.js";
+import Panel from "../../src/Panel.js";
+import Title from "../../src/Title.js";
+import {
+	PANEL_ICON,
+} from "../../src/generated/i18n/i18n-defaults.js";
+import PanelAccessibleRole from "../../src/types/PanelAccessibleRole.js";
+import TitleLevel from "../../src/types/TitleLevel.js";
+
+describe("API", () => {
+	it("Change of header text", () => {
+		const initialHeader = "Initial Header Text";
+		const newHeader = "New Header Text";
+		cy.mount(<Panel headerText={initialHeader}></Panel>);
+
+		cy.get("[ui5-panel]")
+			.shadow()
+			.find(".ui5-panel-header-title")
+			.as("header");
+
+		cy.get("@header")
+			.should("have.text", initialHeader);
+
+		cy.get("[ui5-panel]")
+			.invoke("prop", "headerText", newHeader);
+
+		cy.get("@header")
+			.should("have.text", newHeader);
+	});
+});
+
+describe("Interaction", () => {
+	it("Collapsing fixed panel is not possible", () => {
+		const toggleEventStub = cy.stub();
+		cy.mount(<Panel fixed={true} headerText="Fixed Panel" onToggle={toggleEventStub}>
```

Comment:
```
<Panel onToggle={cy.stub().as("toggleEvent")} />
```

Later on instead of wrapping the variable, you can use the alias:
```
cy.get("@toggleEvent")
```